### PR TITLE
Test refactoring

### PR DESCRIPTION
**Pull request by bquenin**
- Remove `test-*` targets in Makefile
- Factor in the test initialization code
## Verification

```
$ sudo swarm start
$ make test
```

### CODE REVIEW

**Review by freignat91 — APPROVED** [review]
after: rm-rf vendor && glide install
Tested ok

File: Makefile [hunk 1]
```diff
@@ -122,28 +122,12 @@ build-image:
...
-test-logs:
+test:
+#	@go test -v $(REPO)/api/rpc/build
 	@go test -v $(REPO)/api/rpc/logs
-
-test-project:
-	@go test -v $(REPO)/api/rpc/project
-
-test-service:
+#	@go test -v $(REPO)/api/rpc/project
```
Comment (subfuzion) [inline Makefile]: Let's get rid of `-v` and make tests silent. If tests are failing and people want to debug them, they can run the specific test manually with the option (ex: `go test -v github.com/appcelerator/amp/api/rpc/logs`).

Comment (bquenin) [reply Makefile]: Ok

**Review by subfuzion — APPROVED** [review]
👍

**Pull request merged**